assets/javascripts/modules/validatorFocus.js

```diff
@@ -7,10 +7,26 @@
     }
 
     $('.error-summary a').on('click', function(e) {
+      var $this = $(this);
+      var focusId = $this.attr('data-focuses');
+      if (!focusId)
+      {
+        focusId = ("" + $this.attr("href")).trim();
+        if (focusId.indexOf("#") !== 0)
```

Comment:
> It's for backwards compatibility - see the previous code

which backwards compatibility do you mean? theres nothing in the original code? https://github.com/hmrc/assets-frontend/blob/master/assets/javascripts/modules/validatorFocus.js